.github/dependabot.yml

```diff
@@ -9,3 +9,10 @@ updates:
   ignore:
     # Particular.Analyzers updates are distributed via RepoStandards
     - dependency-name: "Particular.Analyzers"
+    # Changing this dependency affects the .NET SDK and Visual Studio version supported by 
+    # a Roslyn analyzer, updates should be more intentional than an automated update
+    - dependency-name: "Microsoft.CodeAnalysis.CSharp.Workspaces"
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
Should we have a time specified here as well?

What about `open-pull-requests-limit`

Reply:
https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#scheduletime

> By default, Dependabot checks for new versions at 05:00 UTC

I'm not sure we need to deviate from the default.

Reply:
Added open-pull-requests-limit. I'm not sure why we specify a time for NuGet honestly, given that we're global and have no "middle of the night."

Reply:
> Added open-pull-requests-limit. I'm not sure why we specify a time for NuGet honestly, given that we're global and have no "middle of the night."

In that case, perhaps we should remove it from the other entry?

Reply:
@adamralph You added the original dependabot file way back in https://github.com/Particular/RepoStandards/commit/5a5e7fa50fc6be268f8bf26b45d41ba02d2ee091#diff-9845e2a184e66c18e4f2dc0ad6eaa83a0ad0ad195bce1ad99df4d27456e86be3 - what do you say, remove the time for NuGet or leave it?

Reply:
I say remove it.